scss/_base_button.scss

```diff
@@ -5,6 +5,10 @@
   button {
     @include button-pattern;
     line-height: 1rem;
+
+    span + & {
+      margin-left: .5rem;
```

Comment:
With `.5rem` of margin-left: https://cl.ly/1k1W0J330q2r

With `1rem` of margin-left: https://cl.ly/2a2S46252z3t